### ISSUE TITLE
Export ignores folders in config folder

### ISSUE THREAD

**Issue by killbot24** [modrinth/code#2285]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [X] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

0.8.2

### What operating systems are you seeing the problem on?

Windows

### Describe the bug

When exporting a modpack and haveing config selected
It ignores/ excludes folders in the file

Given a lot of mods have folders with multi configs kinda breaks it

### Steps to reproduce
1. Get a modpack/create one that has folders in config eg one with a fancy menu
2. click export on a modpack
3. look in config no folders present even if in the modpack folder there is folder in the config folder
4. Export. 

### Expected behaviour

When exporting  a mod pack it should export everything in the config folder

**Comment by MiguVT** [comment]
Same issue, i will edit the zip and move the overrides manually, i dont find other fix at the moment

**Comment by Norbiros** [comment]
Duplicate of  #2271. It was fixed in #2273

**Comment by MiguVT** [comment]
when will it be released in prod?

**Comment by phonkycat** [comment]
> when will it be released in prod?

I would assume soon, appears it has already been merged.

**Comment by DevLung** [comment]
> when will it be released in prod?

it's released now (0.8.5)

**Comment by MiguVT** [comment]
> ### Please confirm the following.
> 
>     * [x]  I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
> 
>     * [x]  I have tried resolving the issue using the [support portal](https://support.modrinth.com)
> 
>     * [x]  I have ensured my Modrinth App installation is up to date
> 
> 
> ### What version of the Modrinth App are you using?
> 
> 0.8.2
> ### What operating systems are you seeing the problem on?
> 
> Windows
> ### Describe the bug
> 
> When exporting a modpack and haveing config selected It ignores/ excludes folders in the file
> 
> Given a lot of mods have folders with multi configs kinda breaks it
> ### Steps to reproduce
> 
>     1. Get a modpack/create one that has folders in config eg one with a fancy menu
> 
>     2. click export on a modpack
> 
>     3. look in config no folders present even if in the modpack folder there is folder in the config folder
> 
>     4. Export.
> 
> 
> ### Expected behaviour
> 
> When exporting a mod pack it should export everything in the config folder
> ### Additional context
> 
> _No response_

You can close the issue, fix released <3

**Comment by killbot24** [comment]
Still occours on 0.8.5

**Comment by MiguVT** [comment]
> Still occours on 0.8.5

Dont happens, try purge cache:
![imagen](https://github.com/user-attachments/assets/8cc2d6f4-7a02-4729-8e29-49ace0996ec4)

**Comment by Felix14-v2** [comment]
If the cache is still can be broken, then the issue is still relevant

**Comment by MiguVT** [comment]
> If the cache is can be broken, then the issue is still relevant

No, it should be other issue, its a known issue that cache bugs some things like modpack updates, some functions when edited, etc.
